x-pack/plugins/security_solution/public/common/hooks/eql/helpers.ts

```diff
@@ -10,55 +10,105 @@ import { inputsModel } from '../../../common/store';
 import { EqlSearchStrategyResponse } from '../../../../../data_enhanced/common';
 import { InspectResponse } from '../../../types';
 import { EqlPreviewResponse, Source } from './types';
-import { EqlSearchResponse } from '../../../../common/detection_engine/types';
+import { BaseHit, EqlSearchResponse } from '../../../../common/detection_engine/types';
 
 type EqlAggBuckets = Record<string, { timestamp: string; total: number }>;
 
 export const EQL_QUERY_EVENT_SIZE = 100;
 
-// Calculates which 2 min bucket segment, event should be
-// sorted into
+/**
+ * Calculates which 2 min bucket segment, event should be sorted into
+ * @param eventTimestamp The event to be bucketed timestamp
+ * @param relativeNow The timestamp we are using to calculate how far from 'now' event occurred
+ */
 export const calculateBucketForHour = (eventTimestamp: number, relativeNow: number): number => {
   const diff: number = relativeNow - eventTimestamp;
   const minutes: number = Math.floor(diff / 60000);
   return Math.ceil(minutes / 2) * 2;
 };
 
-// Calculates which 1 hour bucket segment, event should be
-// sorted into
+/**
+ * Calculates which 1 hour bucket segment, event should be sorted into
+ * @param eventTimestamp The event to be bucketed timestamp
+ * @param relativeNow The timestamp we are using to calculate how far from 'now' event occurred
+ */
 export const calculateBucketForDay = (eventTimestamp: number, relativeNow: number): number => {
   const diff: number = relativeNow - eventTimestamp;
   const minutes: number = Math.floor(diff / 60000);
   return Math.ceil(minutes / 60);
```

Comment:
Optional you can do the same thing here as mentioned above:
https://github.com/elastic/kibana/pull/80750/files#r508003414